### DISCUSSION TITLE
is there a way to run tar_combine from another pipeline?

### DISCUSSION THREAD

**Discussion by pvtodorov** [ropensci/targets#977] [Help] [answered]
Is there a way to build some targets in one pipeline and run `tar_combine` on them in a second pipeline?

I currently have one pipeline which generates a lot of outputs. I would like to check which of these didn't error, match some of their names, etc. I can get the targets' names and states using functions like `tar_meta`, `tar_manifest`, `tar_built`, ... but these are still character types and I can't figure out a way to reconstitute them as actual _targets_ that are required by `tar_combine`.

**Comment by wlandau** [comment, marked as answer]
There are no plans to support chained pipelines, but you can set `tar_option_set(error = "null")` and then write your `tar_combine()` command in a way that is robust to `NULL` objects. (The default `command = vctrs::vec_c(!!!.x)` handles nulls automatically.)

**Reply by pvtodorov** [reply]
thanks this is really helpful! I have set `error = "null"` and no longer need to perform the additional check!
